lua/kulala/parser/inspect.lua

```diff
@@ -15,7 +15,7 @@ M.get_contents = function()
   end
   if req.body ~= nil then
     table.insert(contents, "")
-    local body_as_table = vim.split(req.body, "\r\n")
+    local body_as_table = vim.split(req.body, "\r?\n")
```

Comment:
text as parsed by tree-sitter doesn't include carriage returns. i tested this change with the tree-sitter option disabled as well